src/apps/projinfo.cpp

```diff
@@ -1248,6 +1263,76 @@ int main(int argc, char **argv) {
         }
     }
 
+    if (!listCRS.empty()) {
+        auto params = proj_get_crs_list_parameters_create();
+        bool all_types = false;
+        std::vector<PJ_TYPE> types;
+        std::vector<std::string> auth_names;
+        auto tokens = split(listCRS, ',');
+        for (auto token : tokens) {
+            auto auth_tokens = split(token, '=');
+            if (auth_tokens.size() == 2 && ci_equal(auth_tokens[0], "auth_name")) {
```

Comment:
projinfo has also a ``--authority`` switch. It would probably better to re-use them instead of stuffing it again in crs-filter

Reply:
done